Safety/AttitudeManager/Src/attitudeStateClasses.cpp

```diff
@@ -101,10 +115,24 @@ attitudeState& sensorFusionMode::getInstance()
 
 void PIDloopMode::execute(attitudeManager* attitudeMgr)
 {
-
-    CommandsForAM *PMInstructions = fetchInstructionsMode::GetPMInstructions();
+    CommandsForAM *PMInstructions = nullptr;
     SFOutput_t *SFOutput = sensorFusionMode::GetSFOutput();
 
+    PID_Output_t *pidOut = nullptr;
+    if(fetchInstructionsMode::isAutonomous())
+    {
+        PMInstructions = fetchInstructionsMode::GetPMInstructions();
+        //TODO: Run controls module with PMInstructions
+    }
+    else
+    {
+        PPM_Instructions_t *teleopInstructions = fetchInstructionsMode::GetTeleopInstructions();
+        _PidOutput = getPIDFromControls(teleopInstructions, SFOutput);
+    }
+
+    #ifdef FIXED_WING
+    PMInstructions = fetchInstructionsMode::GetPMInstructions();
```

Comment:
Get PMInstructions is really weird right now since it isn't fully implemented. We might have to comment it out while testing so it compiles until that side of things is implemented